src/kubectl-asset.ts

```diff
@@ -1,4 +1,4 @@
 import * as path from 'path';
 
 export const ASSET_FILE = path.join(__dirname, 'layer.zip');
-export const LAYER_SOURCE_DIR = path.join(__dirname, '..', 'layer', 'Dockerfile');
+export const LAYER_SOURCE_DIR = path.join(__dirname, '..', 'layer');
```

Comment:
Just add a new constant instead of removing the original one. Then we have the freedom to use either in aws-cdk-lib